src/adapters/rubicon.js

```diff
@@ -410,6 +415,41 @@ RubiconAdapter.masSizeOrdering = function(sizes) {
     });
 };
 
+/**
+ * syncEmily
+ * @summary A user sync dependency for the Rubicon Project adapter
+ * When enabled, creates an user sync iframe after a delay once the first auction is complete.
+ * Only fires once except that with each winning creative there will be additional, similar calls to the same service.
+ * @example
+ *  // Config example for Rubicon user sync
+ *  $$PREBID_GLOBAL$$.rubiconGlobals = {
+ *    userSync: {
+ *      enabled: true,
+ *      delay: 1000
+ *    }
+ *  }
+ */
+function syncEmily(hasSynced) {
+  const defaultUserSyncConfig = {
+    enabled: true,
+    delay: 5000
+  };
+  const iframeUrl = 'https://tap-secure.rubiconproject.com/partner/scripts/rubicon/emily.html?rtb_ext=1';
+
+  let publisherUserSyncConfig = $$PREBID_GLOBAL$$.rubiconGlobals && $$PREBID_GLOBAL$$.rubiconGlobals.userSync;
+  // Merge publisher user sync config with the defaults
+  let userSyncConfig = Object.assign(defaultUserSyncConfig, publisherUserSyncConfig);
+
+  // Check that user sync is enabled and that it has not already been triggered - only meant to fire once
+  if (hasSynced || !userSyncConfig.enabled) {
```

Comment:
minor - could check `hasSynced` at the beginning of the function so it doesn't have to go through the config processing each time.